chirpstack/chirp_handlers.go

```diff
@@ -87,7 +88,14 @@ func (handlerContext *Context) PostChirpV3Event(w http.ResponseWriter, r *http.R
 	user := r.Header.Get("TTNMAPPERORG-USER")
 	packetOut.UserId = user // Default header is empty
 
-	log.Print("["+i+"] Network: ", packetOut.NetworkType, "://", packetOut.NetworkAddress)
+	networkID := r.Header.Get("TTNMAPPERORG-NETWORK")
+	if networkID == "" {
```

Comment:
I wonder if we shouldn't return an error here. I know existing integrations will break, but without a networkID the data is anyway not usable.